src/Core/CMakeLists.txt

```diff
@@ -47,7 +51,7 @@ target_include_directories(
 )
 target_link_libraries(
     ${ra_core_target} PUBLIC OpenMeshCore OpenMeshTools Threads::Threads Eigen3::Eigen
-                             cpplocate::cpplocate PRIVATE std::filesystem
+                             cpplocate::cpplocate OpenImageIO PRIVATE std::filesystem
```

Comment:
Private or Public ?, I don't know what is the difference.

Reply:
"All the include directories following PRIVATE will be used for the current target only, i.e., appending the directories to INCLUDE_DIRECTORIES."
I think we only use OpenImageIO in Core, so I have to change to private. Right ?

Reply:
I've in mind a use case where a radium extension will need to access some oiio features directly : we are implementing a ray-tracer that should use the oiio TextureSystem to have proper mip-map management on CPU (among other oiio features). This might require to have the oiio dependency as a PUBLIC dependency of RadiumCore.
And if oiio is exposed in a public Core header, it should be PUBLIC here to have proper transitivity when linking with Core.